pkg/tsdb/stackdriver/stackdriver.go

```diff
@@ -30,9 +30,15 @@ import (
 )
 
 var (
-	slog             log.Logger
-	legendKeyFormat  *regexp.Regexp
-	metricNameFormat *regexp.Regexp
+	slog log.Logger
+)
+
+var (
+	matchAllCap       = regexp.MustCompile("(.)([A-Z][a-z]*)")
+	legendKeyFormat   = regexp.MustCompile(`\{\{\s*(.+?)\s*\}\}`)
+	metricNameFormat  = regexp.MustCompile(`([\w\d_]+)\.googleapis\.com/(.+)`)
```

Comment:
I'm new to both grafana and stackdriver, so I may be missing something... but I believe some metrics are now named like `kubernetes.io/*`. See https://cloud.google.com/monitoring/kubernetes-engine/migration.

Reply:
Thanks for feedback @abhik! I've updated the regex to include kubernetes